OHS-Hosting-Infrastructure/complaint-tracker
# adds hses_api_password to credentials

## Description of change

Adds `hses_api_password` to credentials file

## Acceptance Criteria

The password is encrypted in the credentials file, and is retrievable by team members.

## How to test

Pull down the branch and run:
`EDITOR="vim" bin/rails credentials:edit`, check that the password is there

## Checklist

(deleted because I didn't think any really applied)

## To the Reviewer

This project is using [Conventional Comments](https://conventionalcomments.org/) to give structure
and context to PR comments. Please use these labels in your comments.

* **praise:**
* **nitpick:**
* **suggestion:**
* **issue:**
* **question:**
* **thought:**
* **chore:**
